src/components/Reports/ReportLayout/ReportLayout.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import type { FC, ReactNode } from 'react';
+import { Box, Typography } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import NavReportsList from './NavReportsList';
+
+interface ReportLayoutProps {
+  id: string;
+  title?: string;
+  children?: ReactNode;
+}
+
+const ReportLayout: FC<ReportLayoutProps> = ({ id, title, children }) => {
```

Comment:
I like named exports better as you can't misspell them when importing and VS Code autocomplete seems better. Also I don't think the `index.ts` reexporting is necessary.
```suggestion
export const ReportLayout: FC<ReportLayoutProps> = ({ id, title, children }) => {
```

Reply:
@OzzieOrca I will follow the direction. :)